depthai_demo.py

```diff
@@ -238,7 +245,7 @@ def print_packet_info(packet):
                             cv2.putText(frame, packet.stream_name, (25, 25), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                             cv2.putText(frame, "fps: " + str(frame_count_prev[window_name]), (25, 50), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                         else: # uint16
-                            if args['pointcloud'] and "depth" in stream_names and "rectified_right" in stream_names and right_rectified is not None:
+                            if not is_rpi and args['pointcloud'] and "depth" in stream_names and "rectified_right" in stream_names and right_rectified is not None:
```

Comment:
remove, keep the old. We shouldn't reach here. IF user requests pcl and it's not available we should inform him.

Reply:
I was just returning a print so I changed it here so that it will not break. but sure. will remove it here since will reaise an error above.